backend/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## JAVA
 
-You need java 11 to run the application. You can check you java version by running this command in your terminal: `java -version`.
```

Comment:
@razine-bensari how come you removed this?

Reply:
fixed